packages/synthetix-main/contracts/interfaces/IAccountModule.sol

```diff
@@ -39,6 +39,15 @@ interface IAccountModule {
      */
     function createAccount(uint256 requestedAccountId) external;
 
+    /**
+     * @dev Allows the account token to notify the system when a transfer has occurred
+     *
+     * Requirements:
+     *
+     * - `msg.sender` must be the account token.
+     */
+    function notifyAccountTransfer(address to, uint256 accountId) external;
+
```

Comment:
This is the function in the core system that gets called by the account system whenever an NFT is transferred. The core system uses it to update its cache of who the owner of a particular NFT is.